projects/forms/src/lib/validators/decimals-after-comma/decimals-after-comma.validator.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { FormControl } from '@angular/forms';
+
+import { decimalsAfterCommaValidator } from './decimals-after-comma.validator';
+
+describe('decimalsAfterCommaValidator', () => {
```

Comment:
Ik vind het eerlijk gezegd een beetje moeilijk om deze testen te lezen.
Het is cognitief moeilijk omdat ik de bedoeling van de test niet kan afleiden uit de beschrijving. (In welke cases geeft hij nu geen error => code is moeilijker te begrijpen dan tekst)